pages/project-version/middleware/index.js

```diff
@@ -110,8 +113,51 @@ const canComment = () => (req, res, next) => {
     .catch(next);
 };
 
+const getPreviousVersion = () => (req, res, next) => {
+
+  const versions = orderBy(req.project.versions.filter(pv => pv.id !== req.versionId),
+    v => moment(v.createdAt).format('YYYY-MM-DD'), ['desc']);
```

Comment:
`desc` doesn't need to be an array here

Reply:
also I'm pretty sure you can just orderBy the UTC string without having to call moment and format here

Reply:
done